transaction/src/plan/action/ics20_withdrawal.rs

```diff
@@ -0,0 +1,98 @@
+use ibc::core::ics24_host::identifier::{ChannelId, PortId};
+use penumbra_crypto::{asset, value, Address, Amount, Balance};
+use penumbra_proto::{core::ibc::v1alpha1 as pb, DomainType};
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+use crate::action::Ics20Withdrawal;
+
+/// A planned [`Ics20Withdrawal`](Ics20Withdrawal).
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(try_from = "pb::Ics20WithdrawalPlan", into = "pb::Ics20WithdrawalPlan")]
+pub struct Ics20WithdrawalPlan {
+    pub destination_chain_id: String,
+    pub destination_chain_address: String,
+    pub denom: asset::Denom,
+    pub amount: Amount,
+    pub source_channel: ChannelId,
+    pub timeout_height: u64,
+    pub timeout_timestamp: u64,
+
+    pub return_address: Address,
+}
+
+impl Ics20WithdrawalPlan {
+    // NOTE: these are duplicated from action::Ics20Withdrawal. should they be deduplicated?
```

Comment:
Commented above, but AFAICT the best deduplication would be to just use the action itself as the plan.